crates/primitives/src/lib.rs

```diff
@@ -38,10 +38,14 @@ cfg_if::cfg_if! {
     }
 }
 
-#[cfg(feature = "c-kzg")]
+#[cfg(any(feature = "c-kzg", feature = "kzg-rs"))]
 pub use kzg::{EnvKzgSettings, KzgSettings};
 pub use precompile::*;
 pub use result::*;
 pub use specification::*;
 pub use state::*;
 pub use utilities::*;
+
+#[cfg(feature = "kzg-rs")]
+// silence lint
+use kzg_rs as _;
```

Comment:
Do you think this needs to be? 
`#[cfg(all(features = "c-kzg",feature = "kzg-rs"))]`

I am assuming here but if both are enabled kzg-rs is not used,